docs/concepts/projects/config.md

```diff
@@ -170,7 +170,8 @@ environment. If an environment is not present at the provided path, uv will crea
 
 This option can be used to write to the system Python environment, though it is not recommended.
 `uv sync` will remove extraneous packages from the environment by default and, as such, may leave
-the system in a broken state.
+the system in a broken state. To use the system Python environment, set
+`UV_PROJECT_ENVIRONMENT=/usr/local`.
```

Comment:
This is specific to your system, this path differs depending on the OS

Reply:
Forgive my ignorance, I assumed this would be accurate enough in most practical cases.

How about I change it to clarify that this as an example for an applicable OS?

> For example, on a Debian-based OS, you can use `UV_PROJECT_ENVIRONMENT=/usr/local`.

Reply:
That sounds great, thank you!